packages/client-cli/lib/openapi-common.mjs

```diff
@@ -99,7 +99,14 @@ export function writeOperations (interfacesWriter, mainWriter, operations, { ful
 
 export function writeProperties (writer, blockName, parameters, addedProps) {
   if (parameters.length > 0) {
-    writer.write(`${blockName}: `).block(() => {
+    let allOptionalParams = true
+    parameters.forEach(({ required }) => {
```

Comment:
Could you please use a `for..of` loop instead?

Reply:
Sure thing, done on 86b64509